### ISSUE TITLE
installation problem: No such file or directory: '/usr/local/cuda/bin/nvcc'

### ISSUE THREAD

**Issue by chiliuliu** [microsoft/DeepSpeed#4553]
Hi, I met this problem as shown. But I actually have nvcc in my conda environment. 

conda list |grep nvcc
nvidia-cuda-nvcc-cu11     11.7.99                  pypi_0    pypi

And torch shows GPUs are available in my environment:
code:
#########
import torch

if torch.cuda.is_available():
    print("GPU is available.")
else:
    print("No GPU available.")
num_gpus = torch.cuda.device_count()
print(f"Number of available GPUs: {num_gpus}")
device=torch.cuda.get_device_name(0)
print(f"config.device is {device}")

print(torch.version.cuda)
#########

output:
#########
GPU is available.
Number of available GPUs: 2
config.device is Tesla P100-PCIE-16GB
11.7
#########

So, in this case, How can fix it?

**Comment by mrwyattii** [comment]
@chiliuliu what OS are you using? How are you installing DeepSpeed? Can you share the full output of the error? Thank you

**Comment by chiliuliu** [comment]
> @chiliuliu what OS are you using? How are you installing DeepSpeed? Can you share the full output of the error? Thank you

Hi @mrwyattii, I've solved this problem. 
I used a Ubantu Docker Image for finetuning a LLM. And I found there is always no Nvcc installed in the OS, while DeepSpeed needs it. So, I get to download the Cuda Toolkit from the website:
https://developer.nvidia.com/cuda-downloads

Install all the content except for the drive which is installed before, and Nvcc is found in my container. And DeepSpeed is successfully installed
